vernac/vernacextend.ml

```diff
@@ -232,11 +232,11 @@ let static_vernac_extend ~plugin ~command ?classifier ?entry ext =
     let r = untype_grammar ty in
     let ext = Vernacexpr.{ ext_plugin; ext_entry = command; ext_index = i } in
     let () = vinterp_add depr ext f in
-    let () = Egramml.declare_vernac_command_grammar ~allow_override:false ext entry r in
+    let () = Egramml.declare_vernac_command_grammar ~allow_override:!Flags.in_ml_toplevel ext entry r in
```

Comment:
This seems wrong, why is something getting overridden?

Reply:
Why are we running top_printers multiple times? We should prevent that instead of hacking elsewhere.